File: hearts.py
def printHearts(count):
  if(count > 0):
    for i in range(count):
      for j in range(i+1):
        print("♡", end="")
      print('')
  elif(count < 0):
    count = -count;
    for i in range(count):
      for j in range(count-i):
        print("♡", end="")
      print('')
  else:
    return 0;

while(1):
  print("매개변수를 입력하세요");
  n = int(input())
  exit = printHearts(n);
  if(exit == 0):
    break

File: hearts.js
function printHearts(count) {
  let star = "";

  if (count > 0) {
    for (let i = 0; i < count; i++) {
      for (let j = 0; j < i + 1; j++) {
        star += "♡";
      }
      console.log(star);
      star = "";
    }
  } else {
    count *= -1;
    for (let i = 0; i < count; i++) {
      for (let j = 0; j < count - i; j++) {
        star += "♡";
      }
      console.log(star);
      star = "";
    }
  }
}

printHearts(2);
console.log("-----");
printHearts(5);
console.log("-----");
printHearts(-4);
